### DISCUSSION TITLE
Add support for easily overriding perPage & perPageAccepted options.

### DISCUSSION THREAD

**Discussion by ash123456789** [rappasoft/laravel-livewire-tables#223] [Feature Requests]
Currently there doesn't seem to be an easy way to define custom options for the `$perPage` and `$perPageAccepted` properties. There also doesn't seem to be any documentation about how to do this either, other than just turning it off entirely.

While a default of `10` for `$perPage` and `10, 25, 50` for `$perPageAccepted` are pretty decent ranges, this may change depending on either page or type of application.

I think allowing us to override these properties on the custom livewire class would be beneficial and allow for more configurability.

I'm typing this at almost 1am so please forgive me if I've missed anything 😅

**Comment by rappasoft** [comment]
Greater configurability is on the road map in general, but I'll include this one in the next release today or tomorrow. It will be backwards compatible so ^1.0 should still work fine.

**Reply by rappasoft** [reply]
With 1.2.0 you can set these two on your table and the UI will reflect:

```
public int $perPage = 10;
public array $perPageAccepted = [10, 25, 50];
```
